### PR TITLE
latest cycle support

### PR DESCRIPTION
**Pull request by CohenIdo**
Add latest cycle support. 
update the index pattern constant to a more meaningful name.

### CODE REVIEW

File: x-pack/plugins/cloud_security_posture/common/constants.ts [hunk 1]
```diff
@@ -4,11 +4,11 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-export const CSP_KUBEBEAT_INDEX_NAME = 'logs-k8s_cis*';
+export const CSP_FINDINGS_INDEX_PATTERN = 'logs-k8s_cis*';
```
Comment (CohenIdo) [inline x-pack/plugins/cloud_security_posture/common/constants.ts:7]: Not related to the PR - but the naming here was not right. 
It is a pattern and not a name.

File: x-pack/plugins/cloud_security_posture/common/constants.ts [hunk 2]
```diff
@@ -4,11 +4,11 @@
...
 export const CSP_FINDINGS_INDEX_NAME = 'findings';
 export const STATS_ROUTE_PATH = '/api/csp/stats';
 export const FINDINGS_ROUTE_PATH = '/api/csp/finding';
-export const AGENT_LOGS_INDEX = 'logs-k8s_cis*';
+export const AGENT_LOGS_INDEX_PATTERN = '.logs-k8s_cis.metadata*';
```
Comment (CohenIdo) [inline x-pack/plugins/cloud_security_posture/common/constants.ts:11]: By the latest kubebeat the agent are send to `'.logs-k8s_cis.metadata*'`

File: x-pack/plugins/cloud_security_posture/public/pages/findings/utils.tsx [hunk 3]
```diff
@@ -13,7 +13,7 @@ import type {
   TimeRange,
 } from '../../../../../../src/plugins/data/common';
 import type { CspClientPluginStartDeps } from '../../types';
-import { CSP_KUBEBEAT_INDEX_NAME } from '../../../common/constants';
+import { CSP_FINDINGS_INDEX_PATTERN } from '../../../common/constants';
```
Comment (CohenIdo) [inline x-pack/plugins/cloud_security_posture/public/pages/findings/utils.tsx:16]: Not related to the PR - but the naming here was not right.
It is a pattern and not a name

File: x-pack/plugins/cloud_security_posture/common/constants.ts [hunk 4]
```diff
@@ -8,7 +8,7 @@ export const CSP_KUBEBEAT_INDEX_NAME = 'logs-k8s_cis*';
 export const CSP_FINDINGS_INDEX_NAME = 'findings';
 export const STATS_ROUTE_PATH = '/api/csp/stats';
 export const FINDINGS_ROUTE_PATH = '/api/csp/finding';
-export const AGENT_LOGS_INDEX = 'logs-k8s_cis*';
+export const AGENT_LOGS_INDEX = '.logs-k8s_cis.metadata*';
```
Comment (CohenIdo) [inline x-pack/plugins/cloud_security_posture/common/constants.ts:11]: This is the updated index pattern in the latest kubebeat version

**Review by kfirpeled — APPROVED** [review]

**Pull request merged**